src/gatherers/manifest.js

```diff
@@ -19,30 +19,37 @@
 const Gather = require('./gather');
 const manifestParser = require('../lib/manifest-parser');
 
-/* global document, XMLHttpRequest */
+/* global document, XMLHttpRequest, __inspect */
 
 function getManifestContent() {
+  function post(response) {
+    // __inspect is magically inserted by driver.evaluateAsync
+    __inspect(JSON.stringify(response));
```

Comment:
not bad. so convention would be that you must call `__returnResult(thing)`.  seems OK, yeah.